### ISSUE TITLE
Acunetix XML no longer imports

### ISSUE THREAD

**Issue by daniejstriata** [infobyte/faraday#335]
## Issue Type
 - Bug Report

## Faraday version

Faraday v3.5.0

## Component Name

Acunetix Plugin

## Steps to reproduce

Export WAF XML and import it into Faraday. Nothing happens.

## Expected results

Expect to see the vulnerabilities added found to an existing host.

### Debugging tracebacks (current results)
```

[ INFO ]- 2019-01-29 07:21:10,313 - faraday.ReportProcessor - The file is /home/user/Downloads/20190129_XML_https_xxx.xml, Acunetix
[ DEBUG ]- 2019-01-29 07:21:10,431 - faraday.PluginController - Created plugin_process (140101299657680) for plugin instance (140101773987216)
[ DEBUG ]- 2019-01-29 07:21:10,432 - faraday - ----------------------------------------
[ DEBUG ]- 2019-01-29 07:21:10,432 - faraday - proc_name = Thread-12
[ DEBUG ]- 2019-01-29 07:21:10,432 - faraday - Starting run method on PluginProcess
[ DEBUG ]- 2019-01-29 07:21:10,433 - faraday - parent process: 23447
[ DEBUG ]- 2019-01-29 07:21:10,433 - faraday - process id: 28219
[ DEBUG ]- 2019-01-29 07:21:10,433 - faraday - ----------------------------------------
[ DEBUG ]- 2019-01-29 07:21:10,433 - faraday - Thread-12: New Output
[ DEBUG ]- 2019-01-29 07:21:10,434 - faraday - Plugin raised an exception:
[ DEBUG ]- 2019-01-29 07:21:10,435 - faraday - Traceback (most recent call last):
  File "/home/user/Tools/faraday/plugins/plugin.py", line 407, in run
    self.plugin.processOutput(output)
  File "/home/user/Tools/faraday/plugins/plugin.py", line 134, in processOutput
    self.parseOutputString(output)
  File "/home/user/.faraday/plugins/acunetix/plugin.py", line 254, in parseOutputString
    parser = AcunetixXmlParser(output)
  File "/home/user/.faraday/plugins/acunetix/plugin.py", line 58, in __init__
    self.sites = [data for data in self.get_items(tree)]
  File "/home/user/.faraday/plugins/acunetix/plugin.py", line 85, in get_items
    yield Site(node)
  File "/home/user/.faraday/plugins/acunetix/plugin.py", line 138, in __init__
    self.ip = self.resolve(self.host)
  File "/home/user/.faraday/plugins/acunetix/plugin.py", line 162, in resolve
    '[ERROR] Acunetix XML Plugin: Ip of host unknown ' + host,
TypeError: cannot concatenate 'str' and 'NoneType' objects

[ DEBUG ]- 2019-01-29 07:21:10,443 - faraday - _processAction - 3000 - parameters = ['Acunetix', 123]
[ INFO ]- 2019-01-29 07:21:10,444 - faraday.ModelController - Plugin Started: Acunetix. 
[ DEBUG ]- 2019-01-29 07:21:10,445 - faraday - _processAction - 3001 - parameters = ['Acunetix', 123]
[ INFO ]- 2019-01-29 07:21:10,445 - faraday.ModelController - Plugin Ended: Acunetix

```
Please attach the result of:

pip freeze > requirements_freeze.txt
[requirements_freeze.txt](https://github.com/infobyte/faraday/files/2806178/requirements_freeze.txt)

### Configuration files
Default install
### Reports/Extra data
Acunetix outputs in 3 formats. PDF, HTML and XML.

[20190129_XML_https_eb4_uk_striata_com.xml.zip](https://github.com/infobyte/faraday/files/2806192/20190129_XML_https_eb4_uk_striata_com.xml.zip)
[20190129_Developer_https_eb4_uk_striata_com.html.zip](https://github.com/infobyte/faraday/files/2806194/20190129_Developer_https_eb4_uk_striata_com.html.zip)

### OS

Provide information on your operating system. Example:

$ cat /etc/lsb-release
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=18.3
DISTRIB_CODENAME=sylvia
DISTRIB_DESCRIPTION="Linux Mint 18.3 Sylvia"

**Comment by montive** [comment]
Hi there.

This bug has been fixed. The fix will be release in the upcoming version.

Cheers

**Comment by llazzaro** [comment]
Fixed on v3.6
